var/spack/repos/builtin/packages/mod2c/package.py

```diff
@@ -26,5 +26,5 @@ def cmake_args(self):
             options.append('-DCMAKE_BUILD_WITH_INSTALL_RPATH=1')
         return options
 
-    def setup_environment(self, spack_env, run_env):
-        run_env.set('MODLUNIT', join_path(self.prefix, 'share/nrnunits.lib'))
+    def setup_run_environment(self, env):
+        env.set('MODLUNIT', join_path(self.prefix, 'share/nrnunits.lib'))
```

Comment:
Oops..didn't know this was pending. Anyway, I will remove this package ins separate PR (it's now integrated into coreneuron).

Reply:
This was most likely a leftover, only checked when the file changed :)